requirements/detection.txt

```diff
@@ -1,3 +1,5 @@
 mmpycocotools
+mmcls==0.15.0
```

Comment:
`mmcls` and `timm` are needed for `build.py` and `find.py` ?

Reply:
Agree, why do we need `timm` and `mmcls` in detection task?

Reply:
The `timm` package is needed for [efficientnetv2 MMCV backbone](https://github.com/openvinotoolkit/training_extensions/blob/feature/otx/otx/algorithms/common/adapters/mmcv/models/backbones/efficientnetv2.py). Although we don't use it in detection templates it is called during MMCV models initialization

Reply:
`mmcls` is called during MPA hooks initialization: https://github.com/openvinotoolkit/training_extensions/blob/adc16410ced3ecfb7524cc00d5d5afd76ebaa99c/otx/mpa/modules/hooks/recording_forward_hooks.py#L22

Reply:
hmm, seems strange to me. We need to get rid of this. Because we support stand alone environments for each task - but it is required to install mmcls for detection task.

Reply:
> hmm, seems strange to me. We need to get rid of this. Because we support stand alone environments for each task - but it is required to install mmcls for detection task.

Agree with this opinion, we need to remove this later.

Reply:
As far as I know, `timm` is installed from `mmcls>=0.16.0`. I know that upgrading the mmcls version will solve everything. (we don't need to install timm seperately)
Isn't this a problem that needs to be solved by cleaning up the mpa import? (detection do not use mmcls)

Reply:
@harimkang Thank you for your note, I checked on practice, and `timm` isn't installed with higher `mmcls`. Also, it doesn't have it in [requirements](https://github.com/open-mmlab/mmclassification/tree/master/requirements).
Regarding `mmcls` installation, previously it was installed during `mpa`  installation in [detection requirements](https://github.com/openvinotoolkit/training_extensions/blob/be582a20581d73eacfcb8036f473e4a9f140f951/requirements/detection.txt), so it wasn't so obvious what we need `mmcls` to run detection. 
So yes, import clean up in mpa import can be an option.